050-unidirectional-topic-operator.md

```diff
@@ -0,0 +1,801 @@
+> So long as these subsets are disjoint there is no possibility of conflict in allowing two KafkaTopics in different namespaces to manage the different aspects of a single topic in Kafka.
+> The disjointness could be enforced by a more sophistocated namespace policy which enumerated the configs.
+> The same logic also applies to `spec.partitions` and `spec.replicas`.
+> There would be the possibility of conflict around resource _existence_. 
+> That is, how should the operator behave if the application team delete their KafkaTopic while a KafkaTopic managing the same topic remains in the infra team's namespace?
+> This proposal doesn't attempt to answer that, since it's not proposed to implement it currently.
+> But in order to allow this possibility without introducing an incompatible change in the future we need to define the semantics of fields that are unspecified in the `spec` to mean "unspecified by this CR" rather than "takes the default value".
+
+### Metrics
+
+The BTO exports the following metrics
+
+<table>
+<tr><th>Metric type	<th>Metric name				<th>Status
+<tr><td>counter		<td>strimzi.reconciliations.periodical	<th>Unchanged
+<tr><td>counter		<td>strimzi.reconciliations		<th>Unchanged
+<tr><td>counter		<td>strimzi.reconciliations.failed	<th>Unchanged
+<tr><td>counter		<td>strimzi.reconciliations.successful	<th>Unchanged
+<tr><td>gauge		<td>strimzi.resources			<th>Unchanged
+<tr><td>timer		<td>strimzi.reconciliations.duration	<th>Unchanged
+<tr><td>gauge		<td>strimzi.reconciliations.paused	<th>Unchanged
+<tr><td>counter		<td>strimzi.reconciliations.locked	<th>Unchanged
+<tr><td>gauge		<td>strimzi.resource.state		<th>Unchanged
+</table>
+
+### Emitted Kube `Events`
+
+Let's consider all the cases in which the BTO emits `Event` resources to the Kube apiserver:
+
+* For errors when accessing the topic store. 
+  For UTO there is no topic store, so this cannot happen.
+* For errors when creating partitions. 
+  For UTO ApiExceptions from the Admin client will be reported via `Ready` condition.
+  For other errors application logging and metrics are considered sufficient.
+* For errors when changing the configuration of topics in Kafka
+  For UTO ApiExceptions from the Admin client will be reported via `Ready` condition.
+  For other errors application logging and metrics are considered sufficient.
+* For errors when processing a ZooKeeper znode watch.
+  UTO doesn't use znode watches, so this cannot happen.
+* On attempts to change a `KafkaTopic`'s `spec.topicName`.
+  For UTO this will be reported via the status only.
+* On detection of conflicting changes between Kube and Kafka
+  For UTO such conflicts cannot happen.
+* On attempts to decrease the number of partitions.
+  For UTO this will be reported via the status only.
+
+Thus, the UTO will emit no events.
+Any software which consumed BTO events will not be compatible with UTO.
+
+## Migration
```

Comment:
I think the standalone BTO to UTO migration is quite clear. But the migration within a Kafka cluster would IMHO deserve more details. Will a feature gate drive it? Will there be a flag? Annotation?